addons/godot_pixelorama_importer/pixelorama_file_open/open.gd

```diff
@@ -0,0 +1,70 @@
+extends EditorInspectorPlugin
+
+var tex:StreamTexture
+var user_config_req:bool = true
+var pixel_path:String
+
+var ctl
+
+func _init():
+	#load config file settings
+	var cfg = ConfigFile.new()
+	var err = cfg.load("user://pixel_plugin.ini")
+	if err == 7:
+		cfg.save("user://pixel_plugin.ini")
+		err = cfg.load("user://pixel_plugin.ini")
+	if err == OK:
+		if cfg.has_section_key("location","pixelorama"):
+			pixel_path = cfg.get_value("location","pixelorama")
+			user_config_req = false
+	else:
+		print("Error loading Plugin \n", err)
+
+func can_handle(object):
+	if object is StreamTexture:
+		tex = object
+		return true
+	return false
+
+func parse_end():
+	ctl = preload("inspector_menu.tscn").instance()
+	var btn = ctl.get_node("VBoxContainer/OpenButton")
+	btn.connect("pressed",self,"open_btn_pressed")
+	btn.disabled = user_config_req
+	var win = ctl.get_node("WindowDialog")
+	win.connect("confirmed",self,"_folder_selected")
+	ctl.get_node("VBoxContainer/ConfigButton").connect("pressed",self,"_on_ConfigButton_pressed", [win])
+	add_custom_control(ctl)
+
+func open_btn_pressed():
+	var tex_path = ProjectSettings.globalize_path(tex.resource_path)
+	var extension = ""
+	if OS.get_name() == "X11":
+		extension = ".x86_64"
+	elif OS.get_name() == "Windows":
+		extension = ".exe"
+	else: 
```

Comment:
@OverloadedOrama could you help out for this one? How should we launch Pixelorama for macOS from CLI for editing?

Reply:
Well yes. But since its 2 values for now, I left it as if-else. When I figure out how to do MacOS config, I will change it

Reply:
I am guessing the extension should be ".dmg" and it wouldn't have to check for the .pck file since it's inside the .dmg file, but I'm not sure if this is the correct way. I don't have a macOS machine to test it, unfortunately.

Reply:
Ah alright, cuz yeah we need to get the Pixelorama executable file within the Pixelorama.app folder :P